.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+### General Type of Issue:
+Some example types are:
+- bug
+- enhancement
+- question
```

Comment:
You may consider asking people to prefix their title with one of the following codes:

- ENH: Enhancement, new functionality
- BUG: Bug fix
- DOC: Additions/updates to documentation
- TST: Additions/updates to tests
- BLD: Updates to the build process/scripts
- PERF: Performance improvement
- CLN: Code cleanup

These sort of issue tips can also be hidden in a html comment, e.g. 

```
<!---
your comment goes here
and here
-->
```

Reply:
Thanks for this suggestion. I should have mentioned that we're going to try having separate issue templates for a few different types of issues (using a relatively new GitHub feature). That partially accomplishes what you're suggesting here. For now, I'd like to avoid making users classify their issues more specifically than that, but we might revisit that decision in the future.

I like the suggestion of separating out test-focused issues, and I have created a new label for that purpose.